client/src/views/common/BoostView.tsx

```diff
@@ -10,7 +11,9 @@ import ContentPage from 'components/common/ContentPage';
 export default function BoostView() {
     return (
         <ContentPage>
-            <h1>Hi I&apos;m the Boost View</h1>
+            <div>
```

Comment:
This `div` is unnecessary

Reply:
hahaha true, I don't know what I was thinking